packager/VisibilityChecker.cc

```diff
@@ -451,36 +514,10 @@ class VisibilityCheckerPass final {
         auto importType = this->package.importsPackage(otherPackage);
         if (!importType.has_value()) {
             // We failed to import the package that defines the symbol
-            if (auto e = ctx.beginError(lit.loc, core::errors::Packager::MissingImport)) {
-                auto &pkg = ctx.state.packageDB().getPackageInfo(otherPackage);
-                e.setHeader("`{}` resolves but its package is not imported", lit.symbol.show(ctx));
-                bool isTestImport = otherFile.data(ctx).isPackagedTest() || ctx.file.data(ctx).isPackagedTest();
-                e.addErrorLine(pkg.declLoc(), "Exported from package here");
-                if (auto exp = this->package.addImport(ctx, pkg, isTestImport)) {
-                    e.addAutocorrect(std::move(exp.value()));
-                }
-
-                if (!ctx.file.data(ctx).isPackaged()) {
-                    e.addErrorNote(
-                        "A `{}` file is allowed to define constants outside of the package's namespace,\n    "
-                        "but must still respect its enclosing package's imports.",
-                        "# packaged: false");
-                }
-
-                if (!db.errorHint().empty()) {
-                    e.addErrorNote("{}", db.errorHint());
-                }
-            }
+            imports[lit.symbol].emplace_back(lit.loc);
```

Comment:
In using `lit.symbol` here, this causes another bug.

[package-test-2.zip](https://github.com/sorbet/sorbet/files/14146681/package-test-2.zip)

In this example, the package `B` uses both `A1::Foo` and `A1::Bar`. That causes two `import A1` lines to get added to the `__package.rb` file for `B`.